src/main/java/frc/robot/Commands/EndEffectorCommand.java

```diff
@@ -29,13 +29,28 @@ public void initialize() {
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-    endEffector.setEndEffectorPower(0.5*opController.getRawAxis(5));
+    //TODO add more commands
+    if (opController.getL1Button() && opController.getCrossButton()){
+    endEffector.setEndEffectorPower1(0.5);
+    } else if (opController.getL1Button() && opController.getSquareButton()){
+      endEffector.setEndEffectorPower2(0.5);
+    } else if (opController.getL1Button() && opController.getTriangleButton()){
+      endEffector.setEndEffectorPower1(0.5);
+      endEffector.setEndEffectorPower2(0.5);
+    } else if (opController.getL2Button() && opController.getCrossButton()){
+      endEffector.setEndEffectorPower1(-0.5);
+    } else if (opController.getL2Button() && opController.getSquareButton()){
+      endEffector.setEndEffectorPower2(-0.5); 
+    } else if (opController.getL2Button() && opController.getTriangleButton()){
+      endEffector.setEndEffectorPower1(-0.5);
+      endEffector.setEndEffectorPower2(-0.5);
+    }
```

Comment:
Can you check in with @Benjamin-GRS about our provider patter for controllers?

Reply:
This was written before that.